docs/model_system/model_system.md

```diff
@@ -1,4 +1,209 @@
+
+**Neither `Cell` nor `Symmetry` describes individual atomic sites**. This information lives elsewhere:
+
+```python
+# Individual site symmetries are NOT in the Symmetry section
+# They are derived properties or stored separately:
+
+model_system.wyckoff_sites = ['a8']  # Site-specific Wyckoff assignments
+# Each position has its own local site symmetry (e.g., '4mm', '3m', '1')
+```
+
+**Distinction:**
+- **System symmetry** (in `Symmetry`): *"This crystal has space group Fd-3m"*
+- **Site symmetry** (not in `Symmetry`): *"Atom 1 sits on a site with 4mm point symmetry"*
+- **Coordinate system** (in `Cell`): *"Fractional coordinates are defined relative to these lattice vectors"*
+
+### Integration Example: Silicon Crystal
+
+```python
+si_system = ModelSystem()
+
+# CELL: Defines the coordinate framework
+si_system.cell[0] = Cell(
+    type='conventional',
+    lattice_vectors=[[5.43, 0, 0], [0, 5.43, 0], [0, 0, 5.43]] * ureg.angstrom
+    # → Establishes cubic coordinate system
+)
+
+# POSITIONS: Interpreted within the cell's coordinate system
+si_system.positions = [
+    [0.0, 0.0, 0.0],      # → (0,0,0) in cubic coordinates
+    [0.25, 0.25, 0.25],   # → (1.36,1.36,1.36) Å in Cartesian
+    [0.5, 0.5, 0.0],      # → (2.72,2.72,0) Å in Cartesian
+    # ... 8 atoms total
+] * ureg.angstrom
+
+# SYMMETRY: Describes system-wide symmetry properties
+si_system.symmetry[0] = Symmetry(
+    space_group_number=227,        # System has Fd-3m symmetry
+    point_group_symbol='m-3m',     # System point group
+    crystal_system='cubic',        # System is cubic overall
+    # NOT: individual site symmetries of the 8 atoms
+)
+
+# SITE-LEVEL INFO: Stored separately from system-level symmetry
+si_system.wyckoff_sites = ['a8']  # All 8 atoms are on Wyckoff site 8a
+# Each atom individually has site symmetry '-3m' (not stored in Symmetry section)
+```
+
+**Key insight:** The same `Symmetry` section describes the system regardless of which `Cell` representation is active, because the physical symmetry doesn't change - only the mathematical description of coordinates changes.
+
+## Wyckoff Positions and Symmetry Context
+
+### Wyckoff Site Annotations
+
+`ModelSystem` provides Wyckoff site information through the `wyckoff_sites` quantity:
+
+```python
+model_system.wyckoff_sites = ['a4', 'b2', 'c1']  # Format: letter + multiplicity
+```
+
+**Key features:**
+  - Format: <letter><multiplicity> (e.g., 'a1', 'b2')
+  - Crystallographic standard: Based on International Tables for Crystallography
+  - Geometric classification: Determined by atomic positions only (chemical species ignored)
+  - Per-atom assignment: Each entry corresponds to the position at the same index
+
+  Important: Wyckoff assignments are computed relative to the conventional cell representation, ensuring database compatibility and literature consistency.
+
+  For complete structural uniqueness, combine Wyckoff information with chemical composition data.
```

Comment:
I would suggest that wyckoff sites would have their own model that holds the letter, the multiplicity + any wyckoff  parameters.

Also worth noting that also the primitive and conventional cells might need their own Wyckoff sites if we e.g. want to visualize them. But maybe this is not necessary.

Reply:
So you're suggesting splitting the format here? Is this for convention reasons, or maybe querying?

With "Wyckoff parameters", do you mean of the analysis itself, or the symmetry operations at each site?
The representation of these operations is indeed cell-dependent. The locations should simply match the geometric transformations.

Reply:
With wyckoff parameters, I mean that some Wyckoff positions are parametrized with up to three parameters (x, y, z). Now whether  it is important to know these parameters or not is another question. MatID can give out this information [if you explicitly ask](https://nomad-coe.github.io/matid/docs/reference/symmetryanalyzer/#get_wyckoff_sets_conventional), see and example here: https://nomad-coe.github.io/matid/docs/learn/symmetry-analysis#wyckoff-sets.

If you think that just having the letter+multiplicity is enough, then having a list of values should be fine. But this structure cannot be easily extended later, unlike a list of objects (list of objects does have other problems, like the fact that in ES you require nested queries to combine information from several fields).